cypress/setUp/ngrok.js

```diff
@@ -0,0 +1,13 @@
+const ngrok = require('ngrok')   // eslint-disable-line
+
+async function init (port) {
+  const ngrokUrl = await ngrok.connect(port)
+  process.env.API_EXTENSION_BASE_URL = ngrokUrl
+  return ngrokUrl
```

Comment:
this return value is not used

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/11/commits/e8bd6f6f473c5d12f462fad6d61c8ac4c6a92adb